app/models/history.rb

```diff
@@ -253,9 +255,8 @@ def self.continuous_exposure(history)
     return if field[:old_value] == field[:new_value]
 
     creator = history[:household] == :patient ? 'User' : 'System'
-    formatted_new_value = field[:new_value] ? 'on' : 'off'
-    comment = "#{creator} turned #{formatted_new_value} #{field}#{compose_explanation(history, field)}"
-    create_history(history[:patient], created_by, HISTORY_TYPES[:monitoring_change], comment)
+    comment = "#{creator} turned #{field[:new_value]} #{field[:name]}#{compose_explanation(history, field)}."
```

Comment:
Seems like we can use the `compose_message()` method you wrote here.

Reply:
this one (continuous exposure) and pause notifications have slightly different wording where the message is not "<creator> <verb> <field> from <old_value> to <new_value>"

Reply:
Yeah, I didn't think it would hurt to have them use that wording ("from on to off") or something, but if this is the wording Kelly and them wanted it's fine.